static/templates/actors/character/tabs/crafting.hbs

```diff
@@ -153,7 +157,9 @@
                     {{#if @root.editable}}
                         <li class="formula-header empty">
                             <h4>{{localize "PF2E.FormulaListEmpty"}}</h4>
-                            <button type="button" class="blue" data-action="browse-equipment" data-level="{{lvl}}">
+                            <button type="button" class="blue"
+                              data-action="browse-equipment"
+                              data-level="{{section.level}}">
```

Comment:
```suggestion
                            <button type="button" class="blue" data-action="browse-equipment" data-level="{{section.level}}">
```
Not sure how the indentation occured, or why its 2 instead of 4.